api/public/swagger.yaml

```diff
@@ -57,16 +57,16 @@ components:
                     type: string
                     description: 'Client Secret'
                     example: someRandomCryptoString
-                accessTokenExpiryDays:
+                accessTokenExpiration:
                     type: number
                     format: double
-                    description: 'Number of days in which access token will expire'
-                    example: 1
-                refreshTokenExpiryDays:
+                    description: 'Number of seconds after which access token will expire. Default is 86400 (1 day)'
+                    example: 86400
+                refreshTokenExpiration:
                     type: number
                     format: double
-                    description: 'Number of days in which access token will expire'
-                    example: 30
+                    description: 'Number of seconds after which access token will expire. Default is 2592000 (30 days)'
+                    example: 2592000
```

Comment:
it would be nice (where possible) to provide the logic behind numbers. So instead of `2592000` use `30*24*60*60`